.github/workflows/changelog.yml

```diff
@@ -16,6 +16,10 @@ jobs:
     - name: Get changed files
       id: files
       uses: Ana06/get-changed-files@v1.1
+      # this Action may throw the below error, e.g. when not properly rebased
+      # however, it still gets the modified files and we can continue
+      # Error: The head commit for this pull_request_target event is not ahead of the base commit.
+      continue-on-error: true
```

Comment:
great if they fix it upstream, the arguments in the linked issue make sense. if the fix is ready soon I'm fine with that otherwise we can use this workaround to ease our development

Reply:
I don't know if it will make it upstream. The reason why I released my own version is because the project didn't seem to be maintained. I have released [Ana06/get-changed-files@v1.2](https://github.com/Ana06/get-changed-files/releases/tag/v1.2) which includes the fix. We are already using a fork of upstream, so let use my new version and change it to upstream if they release a version which include the same fixes.